adhocteam/Head-Start-TTADP
# Update the data anonymizer script to include grant and program specialists

## Description of change
This PR includes changes to allow removing PII for the newly added grant and program specialists. It also removes entries from the RequestErrors table


## How to test
- Check that in either in Sandbox, Dev, or Staging databases the grant table doesn't include actual grant and program specialists' information. A check in one of the environments is enough.
For example on sandbox:
```
cf login --sso
cf target -s ttahub-sandbox
cf connect-to-service tta-smarthub-sandbox ttahub-sandbox
\g
select * from "Grants";
```

- Verify that there are no entries in the RequestErrors table.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-492


## Checklists

### Every PR


- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
